docs/GettingStarted.md

```diff
@@ -119,23 +142,126 @@ Check the properties of the created stream. The **format** is one property which
     oboe::AudioFormat format = stream->getFormat();
     LOGI("AudioStream format is %s", oboe::convertToText(format));
 
-Now start the stream. 
+Now start the stream.
 
     stream->requestStart();
 
 At this point you should start receiving callbacks.
 
 ### Closing the stream
-It is important to close your stream when you're not using it to avoid hogging audio resources which other apps could use. To do this use: 
+It is important to close your stream when you're not using it to avoid hogging audio resources which other apps could use. To do this use:
 
     stream->close();
 
 `close()` is a blocking call which also stops the stream.
 
 It is usually advisable to close your stream when [`Activity.onPause()`](https://developer.android.com/guide/components/activities/activity-lifecycle#onpause) is called.
 
+## Using a ManagedStream
```

Comment:
I think for the getting started guide we should recommend a single approach: either a `ManagedStream` or an `AudioStream`. My preference is to use `ManagedStream` as this is simpler.

Additionally, the benefits of using a `ManagedStream` should be explained up front i.e. mentioning that it handles sample rate, format and channel count conversions automatically etc.

Reply:
In regards to the second paragraph, the ManagedStream does not handle conversions automatically, that would be added as part of the flowgraph changes, and would happen in either case. The primary benefits of a ManagedStream fall under memory and lifecycle management.